acceptance/bundle/python/restricted-execution/mutators.py

```diff
@@ -0,0 +1,16 @@
+from dataclasses import replace
+from databricks.bundles.jobs import Job, Task
+from databricks.bundles.core import job_mutator, Bundle
+import os
+
+
+@job_mutator
+def read_envs(bundle: Bundle, job: Job) -> Job:
+    value = os.getenv("SOME_ENV_VAR", "default")
```

Comment:
Do you mean to assert that the environment variables are passed? If not can we just write a constant value to `envs.txt`?

Reply:
Got it, could you please add a one line comment here in that case?